test/integration/framework/apiserver.go

```diff
@@ -56,11 +57,10 @@ func RunControlPlane(t *testing.T, ctx context.Context) (*rest.Config, StopFunc)
 	for _, crd := range crds {
 		t.Logf("Found CRD with name %q", crd.Name)
 	}
-	patchCRDConversion(crds, webhookOpts.URL, webhookOpts.CAPEM)
-	patchCRDServed(crds)
```

Comment:
Wait a minute... I only added this yesterday!

Is it really not needed? If not, then remove the function definition too.

Reply:
Apologies, I was rebasing this morning and did not realize this was recently added.

I think it may not be needed- I see that `served` is set to true for all versions in `patchCRDConversion` function (which was called on the line above) [here](https://github.com/jetstack/cert-manager/blob/master/test/integration/framework/apiserver.go#L101-L104). I will remove this now, but it would be good if you could double check?

Reply:
OMG, that's my fault. I originally added it to `patchCRDConversion` then thought it would be clearer as a separate function and forgot to remove the original place. Sorry about that.

Why does `patchCRDConversion` now return a new slice of CRDs?
It seems superficial because inside the function we're still mutating the original CRD objects.

If these functions are called `patchXYZ` then I think it makes sense if they clearly modify the supplied slice of CRDs.

Reply:
yeah it's a good point about the name and I didn't somehow didn't realize it was modifying those CRDs I think- I will change it back